ocs_ci/framework/main.py

```diff
@@ -66,11 +69,42 @@ def init_ocsci_conf(arguments=None):
             os.path.expanduser(framework.config.RUN['bin_dir'])
         )
         utils.add_path_to_env_path(framework.config.RUN['bin_dir'])
+    print(framework.config)
     check_config_requirements()
 
 
+def init_version_defaults(arguments=None):
+    """
+    Update the config object with version specific defaults
+
+    Args:
+        arguments (list): Arguments for pytest execution
+
+    """
+    print(arguments)
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument(
+        '--ocs-version',
+        default=framework.config.ENV_DATA['current_default_ocs'],
+        choices=framework.config.ENV_DATA['available_ocs_versions']
+    )
+    args, vals = parser.parse_known_args(arguments)
+    ocs_version = args.ocs_version
+    # Read in version specific default file
+    conf_file_name = f"{ocs_version.replace('.','_')}_default_config.yaml"
+    print(conf_file_name)
+    conf_file_path = os.path.join(DEFAULT_CONFIG_PATH, conf_file_name)
+    with open(
+        os.path.abspath(os.path.expanduser(conf_file_path))
+    ) as file_stream:
+        default_config = yaml.safe_load(file_stream)
+        framework.config.update(default_config)
+        framework.config.ENV_DATA['ocs_version'] = ocs_version
+
+
 def main(argv=None):
     arguments = argv or sys.argv[1:]
+    init_version_defaults(arguments)
```

Comment:
Updating the config here with version specific values is why the unit tests are not passing. The unit tests are not expecting default config values to be present at this time. We will either need to come up with a way to configure the version specific values at the [same time as the common ones](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/__init__.py#L38) or alter the unit test to expect the version specific values. I personally think we should see if the former is possible without too much effort.

Reply:
> I personally think we should see if the former is possible without too much effort.

+1